prototype2/adc_readout/chopper_integration_test/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(AdcChopperIntgerationTest_INC
   TDCGenerator.h
 )
 
-create_executable(AdcChopperIntgerationTest)
+create_integration_test_executable(AdcChopperIntgerationTest)
```

Comment:
You should probably talk to @mortenjc about which other targets should be treated as "test executables".